CMakeLists.txt

```diff
@@ -59,7 +58,7 @@ unset(AKTUALIZR_CHECKED_SRCS CACHE)
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
 # find all required libraries
-set(BOOST_COMPONENTS log_setup log filesystem program_options)
+set(BOOST_COMPONENTS log_setup log system filesystem program_options)
```

Comment:
Why did we need to re-add the `system` boost library?

Reply:
Oh, sorry, I forgot to answer to this question in the first place.
I couldn't link aktualizr-check-discovery without it, got linkage error saying that some boost symbol is not found.
I will double check it.